src/data-collector/kount.js

```diff
@@ -92,11 +92,12 @@ Kount.prototype._setupIFrame = function () {
   iframe.style.position = 'fixed';
   iframe.style.left = '-999999px';
   iframe.style.top = '-999999px';
+  iframe.title = IFRAME_ID_PREFIX + this._deviceSessionId;
```

Comment:
Let's not give this a unique id. Since this title is not used by screen readers, its only utility is to be found via automated tests, and giving it something to easily match against would be better.

While we're here, let's add the `aria-hidden` bit too.

```suggestion
  iframe.title = 'Braintree-Kount-iframe';
  iframe.setAttribute('aria-hidden', 'true');
```

Reply:
I agree. I've pushed the changes.